subprojects/core-api/src/main/java/org/gradle/api/configuration/BuildFeatures.java

```diff
@@ -33,14 +33,17 @@
 public interface BuildFeatures {
 
     /**
-     * State of the <a href="https://docs.gradle.org/current/userguide/configuration_cache.html">Configuration Cache</a> feature in the build.
+     * Status of the <a href="https://docs.gradle.org/current/userguide/configuration_cache.html">Configuration Cache</a> feature configuration in the build.
+     * <p>
+     * Note that the status only describes whether the feature is enabled or disabled.
```

Comment:
It also describes whether it was requested or not.

I'd instead say something "Note that this only describes the status of the feature. It does not provide..."